source/sync/data-access-patterns/flexible-sync-permissions-guide.txt

```diff
@@ -217,11 +246,173 @@ This populates the rule expression box with the following:
 To finish, skip :guilabel:`Advanced Configuration`, and then click 
 :guilabel:`Save Changes` to enable Sync with these permissions.
 
+
+.. _restricted-news-feed:
+
+Restricted News Feed
+--------------------
+
+.. note:: Template Available!
```

Comment:
This note should be added to the other examples above.

Reply:
There are no templates available for those above though...